### DISCUSSION TITLE
QEMU command for virtualized macOS 12 on Apple Silicon

### DISCUSSION THREAD

**Discussion by torarnv** [utmapp/UTM#3894] [UTM Help (macOS)] [answered]
Does UTM run its virtualization of macOS 12 on Apple Silicon via QEMU, or does it use Virtualization.framework directly? I'd like to see and tweak the underlying QEMU command if the former is the case.

**Comment by saagarjha** [comment, marked as answer]
macOS guests are run using Virtualization.
